salesagility/SuiteCRM
# Feature/gdpr bulk optin

## Description




- bulk action - confirm opt in

If Confirm opt in enabled:-

Bulk Action send confirm opt in Email from person/Company list view
Send email via email queue / background task
log confirm opt in date sent on email
Person refers to all module of type Person excluding Users
company refers to all module of type Company

## Motivation and Context


GDPR

## How To Test This

- QRR and Rebuild Minified JS files needed.
- Use BULK ACTION on list view to send confirm opt in emails via email queue

## Types of changes


### Final checklist
